### ISSUE TITLE
Regression: JSDoc's markdown-syntax is not highlighting in editor

### ISSUE THREAD

**Issue by madacol** [microsoft/vscode#84626]
Issue Type: <b>Bug</b>

1. Open new file
2. Set language to `Javascript`
3. copy this:

```js
/**
 * test
 * @param e - **Not highlighting**
 */
const fn = e => {}
```
the text `**Not Highliting**` was shown in blue and bold in the editor

## Now:
![image](https://user-images.githubusercontent.com/6379896/68712759-60631580-056a-11ea-92a9-1116b2242c67.png)

VS Code version: Code 1.40.0 (86405ea23e3937316009fc27c9361deee66ffbf5, 2019-11-06T18:14:08.920Z)
OS version: Linux x64 4.15.0-66-generic

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-8500 CPU @ 3.00GHz (6 x 3900)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: disabled_software<br>skia_renderer: disabled_off<br>surface_control: disabled_off<br>surface_synchronization: enabled_on<br>video_decode: unavailable_off<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 2, 2|
|Memory (System)|15.53GB (0.19GB free)|
|Process Argv|--no-sandbox --unity-launch|
|Screen Reader|no|
|VM|57%|
</details><details><summary>Extensions (9)</summary>

Extension|Author (truncated)|Version
---|---|---
npm-intellisense|chr|1.3.0
vscode-eslint|dba|1.9.1
gitlens|eam|10.1.2
vscode-npm-script|eg2|0.3.9
flow-for-vscode|flo|1.5.0
asciidoctor-vscode|joa|2.7.6
python|ms-|2019.10.44104
debugger-for-chrome|msj|4.12.1
vscode-jest|Ort|3.0.2

</details>

**Comment by mjbvz** [comment]
This was an intentional change, see #77990. This functionality now is in this extension: https://marketplace.visualstudio.com/items?itemName=bierner.jsdoc-markdown-highlighting

**Comment by iansan5653** [comment]
@mjbvz This closes #65780 then, correct?
